docs/install.md

```diff
@@ -13,40 +13,53 @@ Install the Dota 2 Workshop Tools.
  * Go to the DLC tab of the window that pops up
  * Check the `INSTALL` box next to `Dota 2 Workshop Tools DLC`
 
-## Addon Tool
+## Setting evrything up
 Open the `node.js command prompt`. Not `Node.js` itself, but the command prompt.
 
-### Install Node Dependencies
-Run the command 
+### Navigate with command promt
+Now we will define 3 paths 
+ * `Dota2 installation folder` is something like `C:\Program Files (x86)\Steam\steamapps\common\dota 2 beta\` or at ant other drive or install location, but certainly ends with `dota 2 beta`
+ * `Dota2 addons folder` is folder named `Addons` in `Intallation folder`, so it looks something like `\dota 2 beta\addons`
```

Comment:
but people can create them, and link after, it is more logical, ill work this out with warpdragon